pkg/pillar/cmd/verifier/handlers.go

```diff
@@ -65,8 +65,6 @@ func (v *verifyHandler) create(ctxArg interface{}, objType string,
 	switch typeName {
 	case "VerifyImageConfig":
 		go runHandler(ctx, objType, key, h1)
-	case "PersistImageConfig":
-		go runPersistHandler(ctx, objType, key, h1)
```

Comment:
I thought you are keeping this for the 'delete the persistent file' case.

Reply:
WIP on that after our discussion.

Reply:
This is not needed anymore as when VolumeMgr deletes a PersistImageStatus, `handlePersistImageStatusDelete()` is invoked by the subscriber which deletes the verified file.

Reply:
This is not needed anymore as when volumeMgr deletes a PersistImageStatus, subscriber in Verifier calls `handlePersistImageStatusDelete()` which deletes the verified file.

Reply:
ok